src/controller/CHIPDeviceController.cpp

```diff
@@ -608,7 +608,7 @@ void DeviceCommissioner::RendezvousCleanup(CHIP_ERROR status)
         mPairingDelegate->OnPairingComplete(status);
     }
 
-    if (mDeviceBeingPaired != kNumMaxActiveDevices)
+    if (mDeviceBeingPaired != kNumMaxActiveDevices && mStorageDelegate != nullptr)
```

Comment:
@pan-apple Please take a look here too?

Reply:
Should we add a TODO here to remove this condition when all the controller apps have implemented the storage delegate? To support multiple device pairings, storage delegate cannot be optional.

Reply:
Sure, I'll add TODO